wdae/wdae/studies/study_wrapper.py

```diff
@@ -231,8 +231,9 @@ def __init__(self, genotype_data_study, pheno_db, gene_scores_db):
     def __getattr__(self, name):
         return getattr(self.genotype_data_study, name)
 
+    @property
     def is_group(self):
-        return self.genotype_data_study.is_group()
+        return self.genotype_data_study.is_group
```

Comment:
It seems like we do not have tests for this method of StudyWrapper. Could you please add some?

Reply:
Added tests